### ISSUE TITLE
Fluent menu list component doesn't scale with text and text values aren't editable within properties panel.

### ISSUE THREAD

**Issue by Counsell1111** [divriots/story-to-design#129]
## Fluent menu list component scale and text value issue ##

Here's the menu list from Fluent, no changes have been made to either the markup or the CSS, this is a bare-bones version of the standard Fluent menu list and here are a few issues keeping it from being usable within Figma.

• Background of menu list component doesn't scale with the component.
![Screenshot 2024-01-11 at 7 29 44 AM](https://github.com/divriots/story-to-design/assets/130077304/5fe2a564-ed5a-4cc1-8a84-c43187674f4d)
• Values of menu list aren't editable within Figma properties pane, we need to click deep into component and edit the actual text of each item.

Bug report identity: Please do not edit

Task path: H5wicg5xvN61lpBgxzah/3NKrCZWygCYQfkbIyx0Z/zxdjv55nm3kn

**Comment by gqio** [comment]
Hello @Counsell1111 

> Values of menu list aren't editable within Figma properties pane, we need to click deep into component and edit the actual text of each item.

For having access to the list of the properties they should be exposed via the args of the stories.

Regarding the depth of 3 levels.

![image](https://github.com/divriots/story-to-design/assets/1063530/6fe8f4d8-6a28-4f56-b3c1-814ef1ccd00b)

Each item has it's own padding, and items are contained in a autolayout container with items padding of 2px. We can't reduce this.

Apparently there is also a padding on the container of the items.
![image](https://github.com/divriots/story-to-design/assets/1063530/f9a66c34-3b84-48d5-96d3-6561a5c577be)
Leading to another level in Figma.

We can't reduce this here.

> • Background of menu list component doesn't scale with the component.

![image](https://github.com/divriots/story-to-design/assets/1063530/92bd7bd6-dccf-4b6a-9410-3b57018fc74a)

Here it works for the background with the official storybook.

Except there is an issue with the padding here when we are reaching the max width of the menu.

I will dig into this issue for the padding right.

Best.

**Comment by gqio** [comment]
Hello @Counsell1111 

Sorry for the delay.

During January and February we revamped completely the algo for autolayout both for story.to.design and html.to.design and now we have been able to fix the last issue with the resizing text and paddings for this issue.

![image](https://github.com/divriots/story-to-design/assets/1063530/b066f658-23de-4fa2-9255-7ae3be31b114)

I will keep you posted when the release is available.

Best.
